pages/index.tsx

```diff
@@ -3,16 +3,16 @@ import styles from '../styles/Home.module.scss';
 // Internal Components
 import NavBar from '../components/NavBar';
 import EventBanner from '../components/EventBanner';
+import HeroBanner from '../components/HeroBanner';
 
 export default function Home() {
   return (
     <>
       <EventBanner contextKey='bannerInfo' />
       <NavBar currentPage='Home' />
       <section className={styles.container}>
-        <aside className={styles.SideArea}></aside>
         <main className={styles.MainArea}>
-          This is where the main page content goes here
+          <HeroBanner imagePath='logo.svg' />
```

Comment:
Controversial and actually goes against what I said earlier, so you can either choose to keep it this way and apply the alt consideration from the comments in `HeroBanner.tsx` or maybe we just embed the image directly in the `HeroBanner.tsx` it'll never change and if it does well just replace the file.